libraries/core_libs/network/src/tarcap/packets_handlers/common/packet_handler.cpp

```diff
@@ -24,6 +30,9 @@ void PacketHandler::processPacket(const PacketData& packet_data) {
       return;
     }
 
+    // Validates packet rlp format
```

Comment:
why don't we call `checkPacketRlpList` here? so you do not need to copy&paste it to every `validatePacketRlpFormat` ?